srcpkgs/goldendict-ng/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'goldendict-ng'
+pkgname=goldendict-ng
+version=24.05.05
+revision=1
+archs="x86_64"
+build_style=cmake
+configure_args="-DUSE_ALTERNATIVE_NAME=ON"
+makedepends="qt6-speech-devel qt6-svg-devel qt6-qt5compat-devel qt6-base-devel qt6-webengine-devel qt6-multimedia-devel qt6-speech-plugin-flite qt6-speech-plugin-mock qt6-speech-plugin-speechd pkg-config ffmpeg-devel hunspell-devel opencc-devel libvorbis-devel xapian-core-devel libeb-devel libzim-devel qt6-tools-devel"
+short_desc="Feature-rich dictionary lookup program"
+maintainer="RandallFlagg <ykpui@hotmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://xiaoyifang.github.io/goldendict-ng/"
+distfiles="https://github.com/xiaoyifang/goldendict-ng/archive/refs/tags/v${version}-${version_name}.tar.gz"
+checksum=e7cf55fef4ddf00d87b4e54489ec33a605b1d5634d0b7c46ddcdd261c3113790
+version_name=LiXia.ecd1138c
```

Comment:
```suggestion
_version_name=LiXia.ecd1138c
```

Reply:
_version_name cannot come after distfiles. It is used by distfiles and the build fails a it is not decalred if it comes after.

Reply:
i never said it had to, it should start with a _ as it's a custom variable